### DISCUSSION TITLE
v0.10.0

### DISCUSSION THREAD

**Discussion by Jonsy13** [litmuschaos/litmusctl#70] [General]
## What's Changed
* Updated APIs as per new schema changes in litmus by @Saranya-jena in https://github.com/litmuschaos/litmusctl/pull/67
* Added proper error messages for upgrade agent by @Saranya-jena in https://github.com/litmuschaos/litmusctl/pull/69

## New Contributors
* @Adarshkumar14 made their first contribution in https://github.com/litmuschaos/litmusctl/pull/59

**Full Changelog**: https://github.com/litmuschaos/litmusctl/compare/v0.9.0...0.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/litmuschaos/litmusctl/releases/tag/0.10.0'>0.10.0</a>.</em>

**Comment by R34SGA** [comment]
Hello Jonsy,
Every Release until today has been tagged vX.X.X and 0.10.0 has been tagged without the "v"  
We would like to know if there's a good reason to that ? 
Thank you for your time, 
Bests regards
Robin  
Chaos TEAM for Pole Emploi with @Vr00mm & @chakirgit

**Reply by R34SGA** [reply]
@Saranya-jena FYI releases should come from tags beginning with "v" to be able for everyone to import in go.mod
we had to do it on our fork with @Vr00mm **[https://github.com/RobinSegura/litmusctl/releases/tag/v0.10.0](https://go.dev/doc/modules/version-numbers)**

**Reply by Jonsy13** [reply]
Hi @RobinSegura Thanks for notifying us about this.
We have created one more tag v0.10.0. Hope this solves the issue.

**Reply by R34SGA** [reply]
Thank you for your quick update :smiley: 
We're still having some issue using litmusctl v0.10.0 go library on litmus 2.9.0 (installed with helm)
maybe @Vr00mm will let you know if we manage to make it work by the end of the week 
thank you again for your time :+1: 
Best regards
